### PR TITLE
Call Cloud HandleLicenseChange when license is changed in Cloud

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
See: https://github.com/mattermost/enterprise/pull/1290

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48563
#### Release Note

```release-note
Changing the license in a cloud environment will now force an update of the subscription and kick the cache for all cluster members
```

### CODE REVIEW

File: api4/cloud.go [hunk 1]
```diff
@@ -630,23 +629,6 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.Err = model.NewAppError("SendCloudWelcomeEmail", "api.user.send_cloud_welcome_email.error", nil, err.Error(), http.StatusInternalServerError)
 			return
 		}
-	case model.EventTypeSubscriptionChanged:
```
Comment (nickmisasi) [inline api4/cloud.go:633]: @neallred We're good to completely get rid of this too then?

Comment (neallred) [reply api4/cloud.go:633]: Yep, given we do the tasks from HandleLicenseChange.

**Review by marianunez — APPROVED** [review]

**Pull request merged**